packages/vm/src/dynamic_link.rs

```diff
@@ -425,7 +428,39 @@ mod tests {
             ));
 
             assert_eq!(result.err().unwrap().message(),
-            "func_info:{module_name:caller, name:foo, signature:[] -> []}, error:Unknown error during call into backend: Some(\"cannot found contract\")"
+            "func_info:{module_name:caller, name:fail, signature:[] -> []}, error:Unknown error during call into backend: Some(\"cannot found contract\")"
```

Comment:
Is it OK that the error is still "Unknown Error"?

Check also line 400.

Reply:
Fixed.

I think L400 is correct because it is returned below.
https://github.com/line/cosmwasm/blob/dynamic_link/packages/vm/src/dynamic_link.rs#L79-L83